src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -246,12 +246,24 @@ public static string GetSubscriptionId(this IEnvironment environment)
             return null;
         }
 
+        /// <summary>
+        /// Gets the name of the currently deployed slot name.
```

Comment:
Is this right? Does it get the slot?

Reply:
This gets the RuntimeSiteName - added this comment after looking back and seeing I had neglected to comment previously. This is a good point though - from my understanding, the runtimeSiteName can indicate that slots exist but might be the name of the site that is currently deployed, and not the name of the slot. I will update.